source/configuring.rst

```diff
@@ -80,38 +113,38 @@ To see the |proxysql| databases and tables use the following commands:
 
 .. seealso::
 
-   |ProxySQL| Documentation: Admin Tables
+   *ProxySQL* Documentation: Admin Tables
       https://github.com/sysown/proxysql/blob/master/doc/admin_tables.md
 
 .. note::
 
-  ProxySQL has 3 areas where the configuration can reside:
+  ProxySQL has three areas where the configuration can reside:
 
   * MEMORY (your current working place)
   * RUNTIME (the production settings)
-  * DISK (durable configuration, saved inside an SQLITE database)
+  * DISK (durable configuration saved in a `SQLite <https://sqlite.org/index.html>`__ database)
 
-  When you change a parameter, you change it in MEMORY area.  That is done by
-  design to allow you to test the changes before pushing to production
-  (RUNTIME), or save them to disk.
+  When you change a parameter, you change it in the MEMORY area.  This method
+  allows you to test the changes before pushing to the change to production
```

Comment:
before pushing to the change -> before pushing the change

Reply:
Fixed.